sample_app/config/initializers/aws.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Aws.config.update(
+  endpoint: ENV['DYNAMO_ENDPOINT'],
+  region: ENV['AWS_REGION']
```

Comment:
This should already be picked up automatically by the SDK - it checks this environment variable if no other source has region.